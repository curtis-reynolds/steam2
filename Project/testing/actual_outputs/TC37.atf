04_validgamename_______sellstd_________buystdwcredit__100.00
00_buystdwcredit___FS_005000.00
