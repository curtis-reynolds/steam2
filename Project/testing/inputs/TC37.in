login
buystdwcredit
buy
validgamename
sellstd
