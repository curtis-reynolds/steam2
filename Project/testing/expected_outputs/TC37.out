Login successful.
Game purchased successfully.
